.travis.yml

```diff
@@ -15,10 +15,10 @@ env:
 matrix:
   fast_finish: true
   include:
+    - node_js: '9'
+      env: TARGET=test-node COVERAGE=true
     - node_js: '8'
       env: TARGET=test-node COVERAGE=true
```

Comment:
we only need to have `COVERAGE=true` on _one_ of these, so please remove it from line 21.